### ISSUE TITLE
Callbacks.ReduceLROnPlateau seems not work for small learning rate

### ISSUE THREAD

**Issue by salan668** [ncullen93/torchsample#110]
Hi, I used torchsample to train the model. However, when I used Adam as the optimizer with 1e-4 learning rate, I found the ReduceLROnPlateau callback seemed not work.

I checked the code in https://github.com/ncullen93/torchsample/blob/21507feb258a25bf6924e4844e578624cda72140/torchsample/callbacks.py#L538 The min learning rate was set to 1e-4, which seemed a little larger than Adam optimizer. I could set the min_lr to `1e-8 - 1e-4` to let it work, but it looked weird. 

So could we set the default value of min_lr to `1e-4` instead of `0.`? So in L538, the code could be changed to `if old_lr > self.min_lr:`

Thank you.
